<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XUPAddFiles</class>
 <widget class="QWidget" name="XUPAddFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Form Class</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbScopes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Scope in which files would be added</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="pTreeComboBox" name="tcbScopes"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbImport">
     <property name="toolTip">
      <string>Imported files will by default be copied to project path unless you provide a relative only path from project path</string>
     </property>
     <property name="title">
      <string>Copy external files to directory</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="cbImport">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pTreeComboBox</class>
   <extends>QComboBox</extends>
   <header>pTreeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>gbScopes</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbImport</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
